spec/support/change_streams/operation.rb

```diff
@@ -50,13 +50,40 @@ def insert_one(coll)
         coll.insert_one(document)
       end
 
+      def update_one(coll)
+        coll.update_one(filter, arguments['update'])
+      end
+
+      def replace_one(coll)
+        coll.replace_one(filter, arguments['replacement'])
+      end
+
+      def delete_one(coll)
+        coll.delete_one(filter)
+      end
+
+      def drop(coll)
+        coll.drop()
```

Comment:
Parentheses are usually elided when a method call has no arguments, i.e. this would normally be spelled `coll.drop`.